### PR TITLE
Missing definitions added

### PR DESCRIPTION
**Pull request by mereolog**
Signed-off-by: mereolog <pawel.garbacz@makolab.com>

Fixes: #83

### CODE REVIEW

File: DE/TrafficIncidents.rdf [hunk 1]
```diff
@@ -938,6 +955,8 @@ A motorcycle conversion assembly makes it possible to convert a conventional two
 				<owl:someValuesFrom rdf:resource="&auto-ti;TrafficIncidentEffect"/>
 			</owl:Restriction>
 		</owl:equivalentClass>
+		<skos:definition xml:lang="en">all road accidents - irrespective of whether there is an accident or serious material damage</skos:definition>
```
Comment (mskulim) [inline DE/TrafficIncidents.rdf:958]: This definition seems unclear. Traffic incident defined as all road accidents?

**Review by mskulim — CHANGES_REQUESTED** [review]
1) There are still classes and properties without definitions. 
2) The names of several properties aren't expressed in lower camel case.

**Comment by mereolog** [comment]
> * There are still classes and properties without definitions.
We will deal with them at a later point.
> * The names of several properties aren't expressed in lower camel case.
This should raised as a separate issue and fixed in a different pull request.

File: DE/TrafficEnvironment.rdf [hunk 2]
```diff
@@ -393,42 +424,50 @@
...
 		<rdfs:subClassOf rdf:resource="&auto-ev;RoadType"/>
 		<rdfs:label xml:lang="en">ordinary road</rdfs:label>
+		<skos:definition xml:lang="en">road that is not a highway</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&auto-ev;OtherSigns">
 		<rdfs:subClassOf rdf:resource="&auto-ev;RoadSurfaceMarking"/>
 		<rdfs:label xml:lang="en">other sign</rdfs:label>
+		<skos:definition xml:lang="en">other sign that can be encountered by the driver on the road</skos:definition>
 	</owl:Class>
```
[the remark this answers is not in the document]
Comment (mereolog) [reply DE/TrafficEnvironment.rdf]: > Class names must be singular.

This should raised as a separate issue and fixed in a different pull request.

File: DE/CarControl.rdf [hunk 3]
```diff
@@ -377,73 +407,87 @@ are referred to collectively as object and event detection and response (OEDR).<
...
 	
 	<owl:Class rdf:about="&auto-cc;Yellow">
 		<rdfs:subClassOf rdf:resource="&auto-cc;TrafficSignalControl"/>
 		<rdfs:label xml:lang="en">yellow</rdfs:label>
+		<skos:definition xml:lang="en">color of a traffic signal, evoked by light with a dominant wavelength of roughly 575-585 nm which is an intermediate between red and green</skos:definition>
 	</owl:Class>
 	
 	<owl:ObjectProperty rdf:about="&auto-cc;approachTo">
 		<rdfs:label xml:lang="en">approach to</rdfs:label>
+		<skos:definition xml:lang="en">relates a vehicle to an intersection it is approaching</skos:definition>
```
[the remark this answers is not in the document]
Comment (mereolog) [reply DE/CarControl.rdf]: > A vehicle can approach not only an intersection, but, for example, another vehicle. Is this definition correct?

This definition was taken from https://www.researchgate.net/publication/318079353_Ontology-Based_Driving_Decision_Making_A_Feasibility_Study_at_Uncontrolled_Intersections, which is the original source of this property.

File: VC/VehicleCore.rdf [hunk 4]
```diff
@@ -1809,11 +1814,11 @@ If you cannot find a suitable identifier in DBPedia, define your own as an insta
 		<sm:directSource rdf:resource="https://schema.org/Motorcycle"/>
 		<rdfs:isDefinedBy rdf:resource="http://purl.org/vso/ns"/>
 		<rdfs:isDefinedBy rdf:resource="https://spec.edmcouncil.org/auto/ontology/VC/VehicleCore/"/>
-		<rdfs:isDefinedBy xml:lang="pl">pojazd samochodowy zaopatrzony w silnik spalinowy, dwukołowy lub z bocznym wózkiem – wielośladowy</rdfs:isDefinedBy>
 		<rdfs:seeAlso rdf:resource="http://dbpedia.org/resource/Motorcycle"/>
 		<rdfs:seeAlso rdf:resource="http://en.wikipedia.org/wiki/Motorcycle"/>
 		<skos:definition xml:lang="en">Any motor vehicle having a seat or saddle for the use of its operator and designed to travel on not more than three wheels in contact with the ground.</skos:definition>
 		<skos:definition xml:lang="en">a single-track, two-wheeled motor vehicle</skos:definition>
+		<skos:definition xml:lang="pl">pojazd samochodowy zaopatrzony w silnik spalinowy, dwukołowy lub z bocznym wózkiem – wielośladowy</skos:definition>
```
[the remark this answers is not in the document]
Comment (mereolog) [reply VC/VehicleCore.rdf]: > Does this class not include electric motorcycles? That's according to the definition in Polish.

This definition was not added in this pull request, so it deserves a separate issue and another pull request.

**Review by mskulim — APPROVED** [review]

**Pull request merged**